fvtest/rastest/cpuLoadAgent.c

```diff
@@ -401,15 +402,17 @@ matrixSquare(void)
 {
 	/* Declare and initialize the matrix. memset works here since this is a true 2D
 	 * array defined in the same scope as the memset. */
-	uintptr_t matrix[MATRIX_SIZE][MATRIX_SIZE];
+	uint32_t size_of_matrix = MATRIX_SIZE * MATRIX_SIZE * sizeof(uintptr_t);
+	uintptr_t *matrix = (uintptr_t*) malloc(size_of_matrix);
 	uintptr_t i = 0;
 	uintptr_t j = 0;
-
```

Comment:
Now that we are using malloc to allocate the memory can we add a NULL check on the result and then just print an error message and return if we fail to allocate?

Reply:
Both of the callers have an OMR_VMTHread so you can pass that through as the parameter.  Once you have the thread you can insert the following macro to get access to the PortLibrary for printing `OMRPORT_ACCESS_FROM_OMRVMTHREAD(vmThread);`

Now that you have access to the PortLibrary you can use the `omrtty_printf` function to write information in the case where we fail to malloc memory.

Reply:
oh and also you should be using the port library `omrmem_allocate_memory` and `omrmem_free_memory` APIs

Reply:
I would change `MATRIX_SIZE` to `MATRIX_ROWS` (or something) and use the symbol `MATRIX_SIZE` to represent `size_of_matrix`.